### PR TITLE
TRITON-2228: Linux CN minimum viable product

### PR DESCRIPTION
**Pull request by bahamat**
make test fails:

```
[root@triton-dev-18 ~/src/sdc-cnapi]# make test
make: *** No rule to make target 'node_modules/nodeunit/bin/nodeunit', needed by 'test'.  Stop.
```

If you `npm install` before `make test` it works. Don't know if this is going to trip up jenkins, but that's not a linuxcn change, so it should be ok I think.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "async": "1.5.2",
     "backoff": "2.5.0",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#ce906fb5ac40b93959c02f7008c3ea7de809dfd6",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#a0936a54a42fc47403a1a3d24d2b3fb7b762b620",
```
Comment (danmcd) [inline package.json:13]: Now that you've pushed sdc-designation this is a stable commit hash, right?

**Review by danmcd — COMMENTED** [review]
One nit, likely to be easy to answer.

File: package.json (hunk 1 above)
Comment (bahamat) [reply package.json:13]: Yes, that's why this one was waiting on sdc-designation. After it was merged I grabbed the hash and added it here. Previously it was just `.../sdc-designation.git#linuxcn`, but now that we're heading back toward master I wanted it to be a specific commit hash again.

**Review by danmcd — APPROVED** [review]
Ship it!

**Pull request merged**